### PR TITLE
refactor(en-docs): add english docs for device and rule_engine

### PR DESCRIPTION
**Pull request by oucb**

### CODE REVIEW

File: en/device/device.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+### Chart data
+
+- For the data reported by the device,  the function point data under the data flow reported by the device can be displayed in the form of chart in real time according to different time dimensions. The chart showing the data can be displayed in a time range of 5 minutes, 1 hour, 6 hours, 1 day, and 1 week. If there are too many function point charts, you can click on the top left corner of the custom chart filter to display the function point data chart you are interested in. All data is real-time data.
+
+  ![device_chart_data](_assets/device_chart_data.png)
+
+
+### Device issue
```
Comment (Zhiw) [inline en/device/device.md]: Device publish
to be discussed

File: en/rule_engine/actions.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  },
+  "value": {
+  }
+}
+```
+
+- action: Containing Webhook address and verification token 
+- value: Rule SQL SELECT result
+
+#### 4. Order issued
```
Comment (Zhiw) [inline en/rule_engine/actions.md]: Device publish or Device control
to be discussed

File: en/device/device.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+The device uplink system can be another device. The sub device list under the device is displayed here , and the list operation is the same as the device list.
+
+
+
+### Device event
+
+- The device uplink message record is the message reported by the device. The page can display the latest reported message of the device in real time without refreshing. To view more data about device events, user can jump to the historical data page to view all the data, and filter and search based on the reporting time and data flow.
+
+  ![device_evnets](_assets/device_evnets.png)
```
Comment (CrazyWisdom) [inline en/device/device.md]: device_evnets.png -> device_events.png

File: en/device/device.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+The connection log of the device is the online or offline information of the connected device. The displayed log information includes the online and offline of the device and the authentication failure.
+
+
+### Sub device
+
+The device uplink system can be another device. The sub device list under the device is displayed here , and the list operation is the same as the device list.
+
+
+
+### Device event
```
Comment (CrazyWisdom) [inline en/device/device.md]: event -> events

File: en/device/device.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+- The device uplink message record is the message reported by the device. The page can display the latest reported message of the device in real time without refreshing. To view more data about device events, user can jump to the historical data page to view all the data, and filter and search based on the reporting time and data flow.
+
+  ![device_evnets](_assets/device_evnets.png)
+
+
+### Functional data
```
Comment (CrazyWisdom) [inline en/device/device.md]: Functional -> capability

File: en/device/device.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+- The page can display the latest parsed information reported by the device without refreshing. The user can also view real-time data and historical data, which can be filtered and searched according to the reporting time and function points.
+
```
Comment (CrazyWisdom) [inline en/device/device.md]: function points -> data points

File: en/device/device.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+### Chart data
+
+- For the data reported by the device,  the function point data under the data flow reported by the device can be displayed in the form of chart in real time according to different time dimensions. The chart showing the data can be displayed in a time range of 5 minutes, 1 hour, 6 hours, 1 day, and 1 week. If there are too many function point charts, you can click on the top left corner of the custom chart filter to display the function point data chart you are interested in. All data is real-time data.
+
+  ![device_chart_data](_assets/device_chart_data.png)
+
+
+### Device issue
+
+- **ActorCloud** or **External Application** issued a message to the device. Click **Add Command** to send a message to the current device either instantly or periodically.
```
Comment (CrazyWisdom) [inline en/device/device.md]: issued a message -> publish a message

File: en/device/gateway.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+When the uplink system of the device is a gateway and the current gateway is selected, a device can be bound to the gateway. The list is the device that is bound to the gateway. The list operation is the same as the device list.
+
+
+### Connection log
+
+The connection log of gateway  is the same as the connection log of the device, that is the online or offline information of the gateway.
+
+
+### Gateway event
```
Comment (CrazyWisdom) [inline en/device/gateway.md]: Gateway event -> Gateway events

File: en/rule_engine/actions.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+Specify topic, and the platform will issue the specified message to the specified device.
```
Comment (CrazyWisdom) [inline en/rule_engine/actions.md]: issue -> publish

File: en/rule_engine/scope_rules.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
+# Scope rule
+
+The scope rule is mainly used for geographic information. User can set a scope for the device as the active area or the prohibited area. When the device exceeds the active area or enters the prohibited area, the rule is triggered.
+
+In general, the scope rule SQL is automatically generated after the user selects it, and can be modified if the user needs it. The scope rules mainly consist of three parts.
+
+**1. Scope type**
+
+- Active area
+-  Prohibited area
```
Comment (CrazyWisdom) [inline en/rule_engine/scope_rules.md]: -  Prohibited area -> - Prohibited area

**Comment by oucb** [comment]
@CrazyWisdom @Zhiw resolved

File: en/device/device.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+- Authentication information: device number, key, and connection user name.
+- Location information: Display device location information on the map;
+- Two tabs in the upper right corner: display the status of the device separately, such as online and offline. The second tab shows the name of the protocol to which the device belongs.
+
+Click the device basic information->location information, where the edit button in the upper right corner can be used to modify the related information. The  subordinate products of basic information cannot be modified, and the information that cannot be modified between different protocols may be different. Location information can open the map and drag the pin to select the location or search the location. It can also automatically define the corresponding location after entering longitude and latitude:
+
+![select_location](_assets/select_location.png)
+
+
+### Connection log
```
Comment (CrazyWisdom) [inline en/device/device.md]: Connection logs

**Pull request merged**